prac_7/myguitars.py

```diff
@@ -0,0 +1,50 @@
+from prac_6.guitar import Guitar
+
+FILENAME = "guitars.csv"
+
+
+def main():
+    """Read file of Guitar details, save as objects, display and save."""
+    print("\nList of Guitars:\n")
+    guitars = load_guitars()
+    for guitar in guitars:
+        print(guitar)
+
+    print("\nAdd a Guitar!\n")
+
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        add_guitar = Guitar(name, year, cost)
+        print(f"{add_guitar}\n")
+        guitars.append(add_guitar)
+        name = input("Name: ")
+    print("\n...snip...\n")
+    print("These are my guitars:")
+
+    for guitar in guitars:
+        print(guitar)
+    save_guitars(guitars)
+
+
+def load_guitars():
+    """Load file of Guitars as a list of objects."""
+    guitars = []
+    in_file = open(FILENAME, 'r')
```

Comment:
Could take FILENAME in as a local variable, this would allow the function to be used for other files.